stac_ipyleaflet/stac_discovery/stac_widget.py

```diff
@@ -1,6 +1,6 @@
 from ipywidgets import VBox, Output, Layout, Text, ToggleButtons, Dropdown, DatePicker, HBox, Textarea, Checkbox
 import pandas as pd
-import ipyevents
+from importlib.resources import files
```

Comment:
`importlib.resources.files` was introduced in python 3.9

if you want the library to support python 3.8 you need to add a dependency like https://github.com/cogeotiff/rio-tiler/blob/dev/pyproject.toml#L34

and then do something like https://github.com/cogeotiff/rio-tiler/blob/dev/rio_tiler/colormap.py#L24-L28

Reply:
Perhaps a question better for @abarciauskas-bgse or @wildintellect - do we need to support Python 3.8?

Reply:
I think it's ok to put a min requirement of Python 3.9